controllers/content_items_controller.go

```diff
@@ -13,35 +15,59 @@ type ContentItemsController struct {
 	arbiter           *urlarbiter.URLArbiter
 	liveContentStore  *contentstore.ContentStoreClient
 	draftContentStore *contentstore.ContentStoreClient
+	errbitNotifier    errornotifier.Notifier
```

Comment:
Picky point: This property would probably be better named `errorNotifier`.

Related:  it's probably worth looking at the [gorename tool](https://godoc.org/golang.org/x/tools/cmd/gorename).  Having said that, it does require you to be working within a `GOPATH`.